### ISSUE TITLE
Audit the strings used for the scores

### ISSUE THREAD

**Issue by afercia** [Yoast/YoastSEO.js#2104]
Noticed while working on https://github.com/Yoast/wordpress-seo/issues/5145 / https://github.com/Yoast/wordpress-seo/pull/12100

Some of the strings provided by YoastSEO.js still use the wording "Bad SEO score". However, we've changed this wording to "Needs improvements" since a while.

These strings are still used in the UI, mostly for visually hidden text. For example: in the traffic light and in the score icon in the admin bar:

<img width="477" alt="screenshot 2019-01-24 at 17 28 30" src="https://user-images.githubusercontent.com/1682452/51695886-6b14e400-2004-11e9-8cc5-0c8ad229af05.png">

https://github.com/Yoast/YoastSEO.js/blob/044e7d5dee692fedc668c2f16be1e052267a34bb/src/config/presenter.js#L14-L19

"Bad SEO score" should be changed to "Needs improvements". 

Not sure about "Feedback", as in the plugin the cases NO_FOCUS and NO_INDEX use separated strings, with some inconsistencies too. See for example in `wordpress-seo/inc/class-wpseo-rank.php`
